eng/versioning/version_client.txt

```diff
@@ -176,6 +176,7 @@ com.microsoft:microsoft-opentelemetry-exporter-azuremonitor;1.0.0-beta.1;1.0.0-b
 # Format;
 # unreleased_<groupId>:<artifactId>;dependency-version
 # note: The unreleased dependencies will not be manipulated with the automatic PR creation code.
+unreleased_com.azure:azure-core;1.11.0-beta.1
 unreleased_com.azure:azure-core-experimental;1.0.0-beta.9
```

Comment:
Are we able to remove the unreleased `azure-core-experimental` tag with `BinaryData` moving into `azure-core`?

Reply:
I think this is an old entry. Even Service Bus is not using unreleased experimental version. So, it can be removed.